src/Parser.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace CSVParser;
+
+class Parser {
+
+    /*
+     * @var $path the file path to the csv file to be parsed.

```

Comment:
Closer to the expected format and I'm not exactly sure how intellisense would handle all the `@var`'s in a single comment this way but generally each `@var` is in it's own comment block just about the variable it is describing. That will also reduce cognitive friction for the developers reading your code.